docs/src/clusters.md

```diff
@@ -68,10 +68,12 @@ solana config set --url https://testnet.solana.com
 
 ##### Example `solana-validator` command-line
 ```bash
+$ trusted_validators=(--trusted-validator 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on)  # testnet.solana.com
+$ trusted_validators+=(--trusted-validator Ft5fbkqNa76vnsjYNwjDZUXoTWpP7VYm3mtsaQckQADN) # Certus One
 $ solana-validator \
     --identity ~/validator-keypair.json \
     --vote-account ~/vote-account-keypair.json \
-    --trusted-validator 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on \
+    "${trusted_validators[@]}" \
```

Comment:
I suspect that this might confuse some entry-level validators - why the array?

Reply:
I wanted to associate a name with each of the trusted validators and bash won't let me to:
```bash
blah \ # invalid comment.  💔 
  blah

Reply:
Maybe I'll just create a table instead